src/components/Bookform.js

```diff
@@ -1,23 +1,53 @@
 /* eslint no-unused-vars: 0 no-undef: 0 */
-import React from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/books';
 
 const BookForm = () => {
```

Comment:
- [ ] This component should have a local state, I would recommend using `useState()` with `title` and `author`, they should have their inputs that updates the local state every time they update (`onChange`) 😸 

    - Remember to follow all the mandatory project requirements 💯 
    ![image](https://user-images.githubusercontent.com/60789317/136591625-d95effc0-539e-4ebc-aa56-8765cf9e8a15.png)